core-bundle/src/Resources/contao/widgets/ImageSize.php

```diff
@@ -103,6 +103,10 @@ protected function validator($varInput)
 			$varInput[0] = parent::validator($varInput[0]);
 			$varInput[1] = parent::validator($varInput[1]);
 		}
+		elseif ($this->mandatory)
+		{
+			$this->mandatory = (bool) $varInput[2];
+		}
```

Comment:
What does this code do? 
The issue seems to be fixed by the change in line 78, these lines here do not seem to be needed?